README.md

```diff
@@ -251,15 +251,13 @@ and break down problems into small chunks.
 @dwyl we write code to make useful things for real people. <br />
 <!-- We don't write code for hedge funds and high frequency traders
 to [help rich people get richer](https://youtu.be/_HDFegpX5gI). -->
-We do not expect that _all_ the people
+We do not expect _all_ the people
 doing the Apprenticeship to write code for their _career_.
-In fact we expect people to gravitate
-to the area of creativity they most enjoy
-and is most fulfilling/valuable to them.
-In the same way that reading, writing and math
-are required in all schools
-(_and are widely considered to be "basic literacy"_)
-we feel that _knowing_ how to code is _essential_.
+But technology and code are at the _centre of what we do @dwyl_
+(as well as the world we live in) and we have found that
+knowing how to code is an ***essential skill***
+to **working effectively** here.
```

Comment:
The _reason_ I was trying to _avoid_ making coding an "_essential_" skill (_even though I **agree* that is should be_) is that we will have people who's **_one_ and _only_ job** will be to Capture and Edit video; which requires _zero_ coding.

Reply:
It requires zero coding, but we still want them to have had some coding _experience_ so that they can empathise well with those _in_ the video.

Reply:
Agreed. Having _empathy_ is _essential_. 👍